src/uucore/src/lib/features/proc_info.rs

```diff
@@ -271,8 +311,32 @@ impl ProcessInformation {
 
         Teletype::Unknown
     }
-}
 
+    pub fn thread_ids(&mut self) -> Rc<Vec<usize>> {
+        if let Some(c) = &self.cached_thread_ids {
+            return Rc::clone(c);
+        }
+
+        let thread_ids_dir = format!("/proc/{}/task", self.pid);
+        let result = Rc::new(
+            WalkDir::new(thread_ids_dir)
+                .min_depth(1)
+                .max_depth(1)
+                .follow_links(false)
+                .into_iter()
+                .flatten()
+                .flat_map(|it| {
+                    it.path()
+                        .file_name()
+                        .map(|it| it.to_str().unwrap().parse::<usize>().unwrap())
```

Comment:
Are you sure about the unwrap?

Reply:
Indeed it's better to just ignore non-numeric directories for future proofing if the kernel starts placing non-numeric directories in /proc/<pid>/task/. I'll change that.

Reply:
Changed